app/wizards/schools/register_ect_wizard/wizard.rb

```diff
@@ -10,8 +10,14 @@ class Wizard < DfE::Wizard::Base
           {
             already_active_at_school: AlreadyActiveAtSchoolStep,
             cannot_register_ect: CannotRegisterECTStep,
+            change_email_address: ChangeEmailAddressStep,
+            change_independent_school_appropriate_body: ChangeIndependentSchoolAppropriateBodyStep,
             change_lead_provider: ChangeLeadProviderStep,
             change_programme_type: ChangeProgrammeTypeStep,
+            change_review_ect_details: ChangeReviewECTDetailsStep,
```

Comment:
Maybe `change_ect_details` makes more sense?

Reply:
Indeed it reads better, but all `change` steps follow the existing step names, prefixed with "change_", ensuring a consistent and predictable pattern.

For example:
- Current step: `review_ect_details: ReviewECTDetailsStep`
- Equivalent change step: `change_review_ect_details: ChangeReviewECTDetailsStep`

I don't have a strong opinion on this, so I'm happy to change it if we still want it.

Reply:
All of them but this one in the mentor journey.